pkg/component/base/execution.go

```diff
@@ -333,7 +367,7 @@ func Validate(data *structpb.Struct, jsonSchema string, target string) error {
 // initialization.
 const SecretKeyword = "__INSTILL_SECRET"
 
-// NewUnresolvedCredential returns an end-user error signaling that the
+// NewUnresolvedCredential returns an end-user error signalling that the
```

Comment:
I'd used the US spelling intentionally as it seems to be what we use in the documentation.